### PR TITLE
LG-3076 Update to remove the cancel link from SMS message

### PR DESCRIPTION
**Pull request by amathews-fs**
Update to remove the cancel link from SMS messages for account deletion.

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 es:
   telephony:
     account_reset_cancellation_notice: Votre demande de suppression de votre compte login.gov a été annulée.
```
Comment (porta-antiporta) [inline config/locales/es.yml:4]: *sigh* will need to update this to be **s**panish, not french.

File: config/locales/fr.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 fr:
   telephony:
     account_reset_cancellation_notice: Votre demande de suppression de votre compte login.gov a été annulée.
-    account_reset_notice: 'Vous avez demandé à supprimer votre compte login.gov. Votre demande sera être traité en 24 heures. Si vous ne souhaitez pas supprimer votre compte, veuillez cancel: %{cancel_link}'
+    account_reset_notice: Comme demandé, votre compte login.gov sera supprimé dans les 24 heures. Vous ne voulez pas supprimer votre compte? Connectez-vous à votre compte login.gov pour annuler.
```
Comment (porta-antiporta) [inline config/locales/fr.yml:5]: I think the last sentence should be either:

> Connectez-vous à votre compte login.gov pour le annuler

or
> Pour annuler, connectez-vous à votre compte login.gov

File: config/locales/es.yml (hunk 1 above)
Comment (porta-antiporta) [reply config/locales/es.yml:4]: oh. i think i see what the problem is, perhaps a past merge error.

`account_reset_cancellation_notice: Su solicitud para eliminar su cuenta de login.gov ha sido cancelada`

Comment (amathews-fs) [reply config/locales/es.yml:4]: Ok. I got it it.

File: config/locales/fr.yml (hunk 2 above)
Comment (amathews-fs) [reply config/locales/fr.yml]: I went with the quicker change for now. I'll see if @juliaelman has a moment to look over the english content before finalizing the other languages.

**Pull request merged**